tests/dataframe/test_intersect.py

```diff
@@ -7,7 +7,7 @@
 def test_simple_intersect(make_df):
     df1 = make_df({"foo": [1, 2, 3]})
     df2 = make_df({"bar": [2, 3, 4]})
-    result = df1.intersect(df2)
+    result = df1.intersect(df2).sort(by="foo")
```

Comment:
Is this intended?

Reply:
It was to fix a flaky test but Colin recently merged a PR in that did this already so no longer needed!